src/translator/translator.h

```diff
@@ -109,6 +109,17 @@ class Translate : public ModelTask {
 
       threadPool.enqueue(task, device, id++);
     }
+
+    if(options_->get<bool>("output-sampling", false)) {
+      if(options_->get<size_t>("beam-size") > 1)
+        LOG(warn,
+            "[warning] Output sampling and beam search (beam-size > 1) are contradictory methods "
```

Comment:
Maybe, remove "highly" in both messages. Other than that good to go and merge.

Reply:
Done